src/common.rkt

```diff
@@ -5,8 +5,9 @@
 (module+ test (require rackunit))
 
 (provide reap
-         flip-lists find-duplicates partial-sums
-         argmins argmaxs index-of set-disjoint?
+         flip-lists drop-at find-duplicates partial-sums
+         argmins argmaxs set-disjoint? subsequence? list-ref* list-set*
```

Comment:
Do we still use `subsequence?`, `list-ref*`, and `list-set*`?

Reply:
We still use `subsequence?` to assert that the `component` is a subsequence of the `variables` in `instruction->operator`, but I just got rid of `list-set*` and `list-ref*`. I think it's a good idea to keep the assertion around so we can find out about it if I got my `connected-components` or `find-preprocessing` implementations wrong, but if we want I can toss that too and get rid of `subsequence?` as well.

Reply:
Ah never mind I just had to add `list-set*` back. See comment below.